.github/workflows/tests.yml

```diff
@@ -92,6 +100,7 @@ jobs:
 
       - name: Run tests
         run: pytest carsus ${{ env.PYTEST_FLAGS }}
+             --arraydiff --arraydiff-reference-path=carsus-refdata/arraydiff  
```

Comment:
Add those lines to the flags variable

Reply:
Done in https://github.com/tardis-sn/carsus/pull/311/commits/2dca529e05d5563f75d0403e060a4348acffe7ed